### ISSUE TITLE
Implement something without rebuilding every time with pip

### ISSUE THREAD

**Issue by zexuanxu** [ALTEMIS-DOE/climate-resilience#1]
Testing the issue capabilities here. 
Also, I was wondering if we can try to do something without rebuilding with pip every time when we make any changes in the code. Since we are still developing it, I think it makes sense to work on the dev set, and try to let us work on code directly. 
Once we have some major updates or make sure the changes are more or less mature, we can push a change to update the pip version.

Does it make sense to you?

**Comment by satyarth934** [comment]
Yeah, I agree with you. We need to do that since we are expecting an active development on this package.
There is a way to automate the PyPI upload process.
Sharing the [automation link](https://jacobtomlinson.dev/posts/2021/automating-releases-of-python-packages-with-github-actions/) here just for reference. 
I'll try and set this up.

**Comment by satyarth934** [comment]
Commenting for personal reference. 
* [ ] Check Github actions
